### DISCUSSION TITLE
EMGU CV on nVidia Jetpack

### DISCUSSION THREAD

**Discussion by ofer** [emgucv/emgucv#674] [Q&A]
Hello,

I'm trying to cross compile EMGU CV to aarch64 but I can't find instructions.  I've built the EMGU openCV version but I can't find where the cvextern library is built so I can cross compile it.  The release versions all come with ubuntu x64 so that's not working for me.

How do I cross compile just cvextern?

**Comment by ofer** [comment]
I ended up adding a second disk on the nVidia Xavier so I could compile it without running out of space, but cross compiling is a much more elegant solution (why should I make an embedded system a build machine?)

**Comment by emgucv** [comment]
You are welcome to contribute scripts for cross-compilation. 

For compilation to target Ubuntu aarch 64, you may also use a ubuntu docker image on Mac OS M1. It is not tested for aarch 64, but the same script for Ubuntu 20.04 (which we use to build the ubuntu 20.04 x64 nuget package) should also work for aarch 64:
https://github.com/emgucv/emgucv/tree/master/platforms/ubuntu/20.04
